test/unit/Makefile

```diff
@@ -1,6 +1,6 @@
 BINDIR := bin
 BINS=$(patsubst %.c,$(BINDIR)/%,$(wildcard *.c))
-LDFLAGS+=$(shell pkg-config --libs r_core)
+LDFLAGS+=$(shell pkg-config --libs r_core) -lm
```

Comment:
uhm maybe -lm should be part of the output from pkg-configi --libs r_core

Reply:
moved it